### DISCUSSION TITLE
Missing Aggregate field in schema.

### DISCUSSION THREAD

**Discussion by Revitate** [omar-dulaimi/prisma-trpc-generator#23] [Q&A] [answered]
From prisma document (https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#aggregate)
and generated schema <img width="801" alt="image" src="https://user-images.githubusercontent.com/19814112/183219825-2d78d35d-8f27-4cf0-acd8-f55582023987.png">
I think it has missing field `_count`, `_avg`, `_sum`, `_min` and `_max` in InputSchema.
Can you help me?

**Comment by omar-dulaimi** [comment]
@Revitate Could you provide a Prisma schema for which this issue occurs?

**Reply by Revitate** [reply]
```
model Admin {
  // Base Schema
  id        Int      @id @default(autoincrement())
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt

  prefix                  String?
  firstname               String?
  lastname                String?
  nickname                String?
  role                    Role    @default(ADMIN)
  username                String  @unique
  password                String
  picture                 String?
  pharmacistLicenceNumber String?
}

enum Role {
  ADMIN
}

generator trpc {
  provider       = "prisma-trpc-generator"
  withMiddleware = false
  withShield     = false
  contextPath    = "../../../context"
  output         = "../src/generated/trpc"
}
```

prisma-trpc-generator 0.4.0

**Comment by omar-dulaimi** [comment, marked as answer]
This is a bug indeed. I have opened an issue here: https://github.com/omar-dulaimi/prisma-trpc-generator/issues/24
